Source/Csla.Shared/Server/NullCriteria.cs

```diff
@@ -0,0 +1,25 @@
+﻿//-----------------------------------------------------------------------
+// <copyright file="EmptyCriteria.cs" company="Marimer LLC">
+//     Copyright (c) Marimer LLC. All rights reserved.
+//     Website: https://cslanet.com
+// </copyright>
+// <summary>Empty criteria used by the data portal as a</summary>
+//-----------------------------------------------------------------------
+using System;
+
+namespace Csla.Server
+{
+  /// <summary>
+  /// Empty criteria used by the data portal as a
+  /// placeholder for a create/fetch request that
+  /// has no criteria.
+  /// </summary>
+  [Serializable]
+  public class EmptyCriteria : Csla.Core.MobileObject
+  {
+    /// <summary>
+    /// Gets an instance of EmptyCriteria
+    /// </summary>
+    public static EmptyCriteria Instance { get; } = new EmptyCriteria();
```

Comment:
If the intent is to have only one instance of `EmptyCriteria` for an app, then I'd make the class `sealed` and give it a private constructor. Otherwise, I'm not sure why `Instance` is here, as a user could just do `new EmptyCriteria()`.

Reply:
Nice improvement - will do (and to the new `NullCriteria` class too.